repair/row_level.cc

```diff
@@ -1844,6 +1843,18 @@ class repair_meta {
     }
 };
 
+// The repair_metas created passively, i.e., local node is the repair follower.
+static thread_local std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>> _repair_metas;
+std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>>& repair_meta::repair_meta_map() {
+    return _repair_metas;
+}
+
+// The repair_metas created actively, i.e., local node is the repair master.
+static thread_local std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>> _repair_metas_masters;
+std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>>& repair_meta::repair_meta_map_masters() {
+    return _repair_metas_masters;
+}
+
```

Comment:
We often keep such debug information in the debug namespace, consider doing the same.

Reply:
Also note that using a map has no advantage while debugging, we don't have the hash or the less functions available (or runnable) so the way to find anything in them is to iterate over all items. You might as well just use a simple `std::list` or `boost::intrusive::list`.

Reply:
This is a good point.  Using list is simpler but it is slower to remove the items. We are not creating those objects on the fast path. I think it is fine.

Reply:
Removing is updating 2 pointers, I don't think it gets much cheaper than that. :)

Reply:
We need to search the repair_meta belongs to a specific node_repair_meta_id when we remove it.